pallets/marketplace/src/tests/withdraw_offer.rs

```diff
@@ -134,3 +134,35 @@ fn nft_owner_should_not_have_rights_for_withdrawing_when_nft_is_burned() {
 			);
 		});
 }
+
+#[test]
+fn withdraw_offer_should_work_when_nft_is_burned() {
```

Comment:
if I am not mistaken, it is the same test as the previous `nft_owner_should_not_have_rights_for_withdrawing_when_nft_is_burned`, and the difference is that here the offer is withdrawn by the offer maker and not by the owner, right?

if so, we could call this test as follows, just to be in line with the previous test name:
`nft_offer_maker_should_have_rights_for_withdrawing_when_nft_is_burned`

Reply:
yes. And in this case there is no error and the offer is actually withdrawn and deposit returned.